lib/kafo/puppet_log_parser.rb

```diff
@@ -18,6 +18,10 @@ def parse(line)
                             [@last_level.nil? ? :info : @last_level, line]
                         end
 
+      if message.include?('Loading facts')
```

Comment:
Is there a possibility of a false positive here? For example something like `error loading facts` getting caught and then hidden away at debug level could be bad.

I think we could scope it to affect only messages that already have level `INFO` then we aren't counting on case sensitivity to do the trick.